Source/projections/Builder/ICanBuildAndRegisterAProjection.ts

```diff
@@ -9,7 +9,7 @@ import { Cancellation } from '@dolittle/sdk.resilience';
 import { Logger } from 'winston';
 import { IProjections } from '..';
 
-export abstract class ICanBuildAndRegisterAProjection {
+export interface ICanBuildAndRegisterAProjection {
```

Comment:
This probably is fine, but I don't know how this will work with whatever @einari has worked on in regards to reflection

Reply:
Yah should be all fine